manifests/s/SmartSoft/SmartFTP/10.0.3185.0/SmartSoft.SmartFTP.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SmartSoft.SmartFTP
+PackageVersion: 10.0.3185.0
+MinimumOSVersion: 10.0.17763.0
+Installers:
+- InstallerLocale: en-US
+  Platform:
+  - Windows.Desktop
+  Architecture: x64
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://static.smartftp.com/static/Products/Client/10.0.3185.0/x64/SmartFTP.msi
+  InstallerSha256: 957A8DAC3C62BB81DD5FF202FCC3CD036778337AFA12578EE64FFB1C9CF8C076
+  ProductCode: '{5F7964B0-E7DC-44B2-AB2C-43A78B50605B}'
+ManifestType: installer
+ManifestVersion: 1.4.0
+ReleaseDate: 2023-11-14

```

```suggestion
ReleaseDate: 2023-11-14
AppsAndFeaturesEntries:
- UpgradeCode: '{2119EFC7-B9BF-445D-9DB9-A9052E8391C5}'
```
The previous version had this added - should this be included here as well?